feature/signup/src/commonMain/kotlin/team/aliens/dms/kmp/feature/signup/navigation/SignUpNavigation.kt

```
@@ -0,0 +1,68 @@
+            navigateToSignIn = navigateToSignIn,
+            termsUrl = termsUrl,
+        )
+    }
+}
+
+fun NavController.navigateToSignUp() {
+    navigate(NAVIGATION_SIGN_UP)
+}
+
+internal fun NavBackStackEntry.getSignUpData(): SignUpData {
+    val signUpData = arguments?.getString(ResourceKeys.SIGN_UP) ?: throw NullPointerException()
```

detekt

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)